person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
```

Comment:
- [ ] good job making the change the previous CR requested however due to this another error occurred, you are calling a method `is_of_age?` but the method is called `of_age?` please fix this small issue.